db/migrate/20201217223026_add_missing_indexes.rb

```diff
@@ -0,0 +1,14 @@
+class AddMissingIndexes < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
```

Comment:
Was just about to ask if we need to enable this to be run in a transaction but it looks like concurrent indexes must be created outside a transaction 👍

Reply:
yep!  indexes take a while to create and we want to make sure we're not locking the table in the meantime.